modyn/model_storage/internal/grpc/model_storage_grpc_servicer.py

```diff
@@ -1,5 +1,7 @@
 """Model storage GRPC servicer."""
 
+# ruff: noqa: N802  # grpc functions are not snake case
```

Comment:
Why is this only added for the model storage grpc servicer and not the other servicers? (Some leftover from removing ruff changes from this commit?)

Reply:
Not sure, upon manual inspection I didn't find other functions that don't comply with pep8 naming conventions.